specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/examples/updateAmlCompute.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2020-03-01",
+    "parameters": {
+      "location": "eastus",
+      "properties": {
+        "computeType": "AmlCompute",
+        "properties": {
+          "description": "some compute",
```

Comment:
It seems that the definition of `Compute` do not have a property of `description`, please double check and make the changes accordingly. 
This is reported by Model Validation.